packages/blockchain-extension/extension/explorer/environmentExplorer.ts

```diff
@@ -123,13 +124,18 @@ export class BlockchainEnvironmentExplorerProvider implements BlockchainExplorer
                 await vscode.commands.executeCommand('setContext', 'blockchain-environment-connected', true);
                 await vscode.commands.executeCommand('setContext', 'blockchain-runtime-connected', true);
             } else {
+                // check if registry has url, if it does, change context?
```

Comment:
There is no need to leave this as a comment, particularly the 'change context?' part.
You can always say 'change context if ops tool environment connected', or remove the comment altogether,

Reply:
Good point!